pkg/release/externals.go

```diff
@@ -0,0 +1,239 @@
+package release
+
+import (
+	"fmt"
+	"github.com/wellplayedgames/unity-installer/pkg/editor"
+	"path"
+	"strings"
+)
+
+func stringPtr(s string) *string {
+	return &s
+}
+
+func editorDataPath(platform, s string) *string {
+	prefix := "{UNITY_PATH}/Editor/Data/"
+	if platform == "darwin" {
+		prefix = "{UNITY_PATH}"
+	}
+
+	out := prefix
+	if out != "" {
+		out = fmt.Sprintf("%s%s", prefix, s)
+	}
+
+	return stringPtr(out)
```

Comment:
```suggestion
	return &out
```

:upside_down_face: ?